soccer/gameplay/root_play.py

```diff
@@ -192,7 +195,7 @@ def goalie_id(self):
 
     @goalie_id.setter
     def goalie_id(self, value):
-        self._goalie_id = None if value == -1 else value
+        self._goalie_id = value
```

Comment:
@justbuchanan I went through and checked over this, I had forgotten to check for other places that goalie_id might have been using None instead of -1. I have fixed these.

@jgkamat If you are referring to the goalie_id setter I definitely remember that it was necessary to remove that, although I am not 100% certain of the reason now. Having looked through the code again and this time checked for places that goalie_id might be expected to be None, I believe I have taken care of all of the issues that might come up from changing the setter.